app/components/validated-input/component.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
```

Comment:
Is this all new code, or heavily based on something derived from elsewhere?

In the latter case, it's helpful to at *least* link to the original at the top of the file. The even more polite course of action, long term, would be to add a message to this repo saying you used someone else's code, similar to the convention of the `NOTICE` file in one of our other projects:

https://github.com/CenterForOpenScience/osf.io/blob/develop/NOTICE

Most of the stuff we rely on is open source, but every now and then the code we want to use has restrictions. Not a big blocker here, but checking licenses before including things can keep your future employers out of trouble.